sdks/python/apache_beam/internal/dill_pickler.py

```diff
@@ -66,6 +90,12 @@
 
   @dill.register(CodeType)
   def save_code(pickler, obj):
+    co_filename = obj.co_filename
+    # MOE:begin_strip
+    # Use relative paths to make pickling lambdas deterministic across Borg task
```

Comment:
Note that the idea of borg task restarts isn't meaningful generally.

Reply:
Updated